src/ir_Daikin.h

```diff
@@ -1,18 +1,15 @@
 /* Copyright 2016 sillyfrog */
-#ifndef IRDAIKINESP_H_
-#define IRDAIKINESP_H_
+#ifndef IR_DAIKIN_H_
```

Comment:
Is there a reason for the trailing underscore? Similarly: `IR_KELVINATOR_H_` and `IR_MITSUBISHI_H_`.